content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -198,7 +198,7 @@ kubectl get node --selector='!node-role.kubernetes.io/control-plane'
 kubectl get pods --field-selector=status.phase=Running
 
 # Get ExternalIPs of all nodes
-kubectl get nodes -o jsonpath='{.items[*].status.addresses[?(@.type=="ExternalIP")].address}'
+kubectl get nodes -o wide
```

Comment:
In my opinion, the existing example is better: it's a way to get _just_ the ExternalIP information (might be useful for a script).

We wouldn't document `kubectl get nodes -o wide` specially; the mention of `kubectl get pods -o wide` earlier should be a clear enough hint about how to get wide output.

Reply:
you are right @sftim , I am closing this PR since 'kubectl get pods -o wide ' is already mentioned earlier. Thank you for the review.